src/bacnet/basic/service/h_apdu.c

```diff
@@ -542,6 +554,7 @@ static bool apdu_unconfirmed_dcc_disabled(uint8_t service_choice)
         switch (service_choice) {
             case SERVICE_UNCONFIRMED_WHO_IS:
             case SERVICE_UNCONFIRMED_WHO_HAS:
+            case SERVICE_UNCONFIRMED_AUDIT_NOTIFICATION:
```

Comment:
please add `#if (BACNET_PROTOCOL_REVISION >= 20)` around the addition since it was added in protocol-revision 20.

Reply:
Added check for protocol-revision 20.